### ISSUE TITLE
Host switching is too slow

### ISSUE THREAD

**Issue by pelikhan** [lancaster-university/codal-core#64]
Assume 1 host and 1 virtual connected, everything good. Then the host goes away and a new host comes on the bus. This should kind of be immediate but it does not appear to work.

**Comment by jamesadevine** [comment]
What do you mean? It should take two seconds to swap hosts... if this is not the case then I'll investigate

**Comment by pelikhan** [comment]
Let me try again. It seemed longer than 2 seconds but it might have been that I was putting a device in bootloader mode which grounds the line.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________
